source/components/GlobalNavigation/components/GlobalNavigationLink/GlobalNavigationLinkLogOut.js

```diff
@@ -1,31 +1,34 @@
 import React from 'react';
 import { withTranslation } from 'react-i18next';
-import withRedirectUrl from "../../hocs/withRedirectUrl";
+import PropTypes from 'prop-types';
 
-class GlobalNavigationLinkLogOut extends React.Component {
-    render() {
-        const { link, t, redirectUrl, className } = this.props;
+import withRedirectUrl from '../../hocs/withRedirectUrl';
 
-        return (
-            <div id={link.title.key} className={className || ''}>
-                <form method="POST" action={link.href}>
-                    <input
-                        type="hidden"
-                        name={link['param-name']}
-                        value={redirectUrl}
-                    />
-                    {/* wds-sign-out__button will make this look like a link */}
-                    <button
-                        type="submit"
-                        className="wds-sign-out__button"
-                        data-tracking-label={link['tracking-label']}
-                    >
-                        {t(link.title.key)}
-                    </button>
-                </form>
-            </div>
-        );
-    }
-}
+const GlobalNavigationLinkLogOut = ({ link, t, redirectUrl, className }) => (
+    <div id={link.title.key} className={className || ''}>
```

Comment:
should `className` be an optional property?

Reply:
aye!